scripts/pulling_sim4life_images/pull_images.py

```diff
@@ -0,0 +1,180 @@
+import os
```

Comment:
Already super nice python code, cheers to this! i will read and see if I find something but I like that it uses good libraries that are pinned and has some logic separated into functions